name/templates/name/stats.html

```diff
@@ -194,18 +194,27 @@
 {% endblock %}
 
 {% block content %}
-    <button class="btn btn-large disabled btn-block"><h1>There are {{ total_names|intcomma }} total names in the UNT Name App.<br>There are {{ total_links|intcomma }} total links attached to Name records.</h1></button><br>
-    <div class="container text-center" id="dashboard">
-        <div class="btn-toolbar pagination-centered">
-            <div class="btn-group">
-                <input class="btn btn-info" id="b0" value="Names Created per Month" type='button'></input>
-                <input class="btn btn-info" id="b1" value="Running Created Names Total" type='button'></input>
-                <input class="btn btn-info" id="b2" value="Names Edited per Month" type='button'></input>
-                <input class="btn btn-info" id="b3" value="Running Edited Names Total" type='button'></input>
+    <div class="row">
+        <div class="col-sm-12">
+            <div class="jumbotron">
+                <div class="text-center">
+                <h2>There are {{ total_names|intcomma }} total names in the UNT Name App.  There are {{ total_links|intcomma }} total links attached to Name records.</h3></div>
```

Comment:
Yes. Haven't gotten to all of them yet though. I think there is also on on the Landing page. Also need to remove emails.